libs/dh/admin/feature-user-management-new/src/lib/invite/invite.component.ts

```diff
@@ -133,67 +129,32 @@ export class DhInviteUserComponent {
     return !!email && this.knownEmails().includes(email.toUpperCase());
   });
 
-  domainExists = computed((): boolean => {
-    const email = this.emailChanged();
-    return !!email && (this.validDomainQuery.data()?.domainExists ?? false);
-  });
-
   knownEmailsQuery = query(GetKnownEmailsDocument);
-  validDomainQuery = lazyQuery(CheckDomainExistsDocument);
 
   knownEmails = computed(
     () => this.knownEmailsQuery.data()?.knownEmails.map((x) => x.toUpperCase()) ?? []
   );
 
-  isLoadingEmails = computed(() => this.knownEmailsQuery.loading());
-  checkingForAssociatedActors = computed(() => this.checkForAssociatedActors.loading());
-  checkForAssociatedActors = lazyQuery(GetAssociatedActorsDocument);
+  isLoadingEmails = this.knownEmailsQuery.loading;
 
   baseInfo = this.nonNullableFormBuilder.group({
     actorId: ['', Validators.required],
     email: [
       { value: '', disabled: true },
       [Validators.required, Validators.email],
-      [
-        (control) => {
-          if (control.value) {
-            return this.checkForAssociatedActors
-              .query({ variables: { email: control.value } })
-              .then((result) => {
-                const associatedActors = result.data?.associatedActors.actors ?? [];
-
-                const isAlreadyAssociatedToActor = associatedActors?.includes(
-                  this.baseInfo.controls.actorId.value ?? ''
-                );
-
-                return isAlreadyAssociatedToActor ? { userAlreadyAssignedActor: true } : null;
-              });
-          }
-
-          return of(null);
-        },
-        (control) => {
-          if (control.value) {
-            return this.validDomainQuery
-              .query({ variables: { email: control.value } })
-              .then((domainCheck) => {
-                return !domainCheck.data.domainExists ? { domainDoesNotExist: true } : null;
-              });
-          }
-
-          return of(null);
-        },
-      ],
```

Comment:
praise: it was long overdue to move these validators to separate functions. Good work 💯